code/modules/camera/camera.dm

```diff
@@ -324,150 +321,64 @@
 		user.visible_message(SPAN_ALERT("[user] has reactivated [src]!"), SPAN_ALERT("You have reactivated [src]."))
 		add_fingerprint(user)
 
-/obj/machinery/camera/proc/autoposition()
-	// This will eventually be removed after wall cameras are repathed using an UpdatePaths script.
-	switch (src.dir)
-		if (NORTH, SOUTH)
-			src.pixel_x -= 10
-
-		if (EAST, WEST)
-			src.pixel_y += 10
-
-		if (NORTHWEST)
-			src.pixel_y -= 10
-			src.set_dir(WEST)
+/// Adds the minimap component for the camera
+/obj/machinery/camera/proc/add_to_minimap()
+	src.AddComponent(/datum/component/minimap_marker/minimap, src.minimap_types, "camera", name=src.c_tag)
 
-		if (NORTHEAST)
-			src.pixel_y -= 10
-			src.set_dir(EAST)
+SET_UP_DIRECTIONALS(/obj/machinery/camera, OFFSETS_CAMERA)
```

Comment:
All these directionals are just named for their direction in the map editor, not indicating that they are in fact cameras. It's not a big deal but I'd rather fix it now if it is an issue with the generic auto-positioning system.